docs/mobility/_index.md

```diff
@@ -2,11 +2,59 @@
 title: "Mobility"
 linkTitle: "Mobility"
 weight: 460
-simple_list: true
 type: "docs"
 description: "Plan and control your robot's motion with Viam's integrated tools."
 image: "/platform/mobility.svg"
 imageAlt: "A path around obstacles."
-empty_node: true
 menuindent: true
+no_list: true
 ---
+
+The {{< glossary_tooltip term_id="rdk" text="RDK" >}} includes several built-in services that make it easy to move your robot in intelligent ways, whether moving a single arm, a connected system of gantries and arms, or an entire robot on wheels.
+
+The following services work together to coordinate your robot's movement:
+
+## Frame system
+
+{{<imgproc src="/services/icons/frame-system.svg" class="fill alignleft" style="max-width: 190px" declaredimensions=true alt="Frame system icon">}}
```

Comment:
The image from the frame system page might be good here - in small https://docs-test.viam.dev/2515/mobility/frame-system/